catalog/core/catalog-core-solr/pom.xml

```diff
@@ -24,6 +24,12 @@
     <artifactId>catalog-core-solr</artifactId>
     <name>DDF :: Catalog :: Core :: Solr</name>
     <packaging>jar</packaging>
+
+    <properties>
+        <!-- For maven-surefire-plugin -->
+        <surefire.argline.append>-Dlog4j.configuration=log4j-test.properties</surefire.argline.append>
```

Comment:
This is to avoid a Windows surefire issue where too many logs are being printed.

Reply:
❓ Should we conditionally do this only for Windows?

Reply:
I feel that the reduced log level is beneficial in general, since it keeps the logs more concise.